.github/workflows/ci-repo.yml

```diff
@@ -11,71 +11,104 @@ on:
       - "main"
       - "release-*"
 
-  workflow_dispatch:
-
-  workflow_call:
```

Comment:
I suspect we'll need to keep this in order for the nightlies to work.

Reply:
added it back